apps/wallet/src/shared/formatting.ts

```diff
@@ -1,10 +1,27 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import BigNumber from 'bignumber.js';
+
 import type { FormatNumberOptions } from 'react-intl';
 
 export const percentageFormatOptions: FormatNumberOptions = {
     style: 'percent',
     maximumFractionDigits: 2,
     minimumFractionDigits: 0,
 };
+
+export const formatPercentage = (
+    num1: bigint | string | number,
```

Comment:
Very unclear to me what these arguments are supposed to represent. Probably better to use terms like denominator and numerator.